guides/framework/working_with_wix_apis.md

```diff
@@ -102,11 +102,12 @@ import { products } from '@wix/stores';
 
 Once you have everything installed and imported, you need to create a client.
 
-In a dashboard component, you create a client by calling the `createClient()` function, using the `authStrategy()` function to define the authentication method, and include a list of all the modules you will use with the client.
+In a dashboard component, you create a client by calling the `createClient()` function, using the `dashboard.auth()` function to define the authentication method, and include a list of all the modules you will use with the client.
```

Comment:
Do you think you need to mention the dashboard.host() function? I don't know anything about any of this so do whatever you think is best.

Reply:
Good idea. It's spoken about a bit higher in the doc, but I can mention here as well.